whipper/src/main/java/org/whipper/resultmode/CompareResultMode.java

```diff
@@ -50,6 +51,10 @@ private String getFileName(Query q){
         return q.getScenario().getId() + File.separator + "errors_for_" + getName() + File.separator + q.getSuite().getId() + "_" + q.getId();
     }
 
+    private String getExpectedResultFileName(Query q){
+        return q.getSuite().getId() + File.separator + q.getSuite().getId() + "_" + q.getId() + ".xml";
```

Comment:
Currently, it's clear which expected result was used for comparison (in case there is an error and you want to read that file). With this new approach, there is a chance that someone adds override and someone else is not aware of it.
I wonder, could we somehow add information which expected result directory was used for comparison? Something easily parsable from either file path or file name (I prefer file name).
Moreover, this will help tools modify proper file if user wants to update expected result (and potentially fail if expected results directory does not exist instead of displaying and modifying wrong expected result).
What do you think?

Reply:
What about adding the information about expected result directory to _error.xml file?

Reply:
Not very visible for user, but probably best option as those files should be read by tools.
I am OK with that.